menas/src/main/scala/za/co/absa/enceladus/menas/utils/converters/SparkMenasSchemaConvertor.scala

```diff
@@ -98,15 +98,17 @@ class SparkMenasSchemaConvertor @Autowired()(val objMapper: ObjectMapper) {
   }
 
   def convertSparkToMenasFields(sparkFields: Seq[StructField]): Seq[SchemaField] = {
-    convertSparkToMenasFields(sparkFields, "");
+    convertSparkToMenasFields(sparkFields, "")
   }
 
   /** Converts a seq of spark struct fields to menas representation */
   def convertSparkToMenasFields(sparkFields: Seq[StructField], path: String): Seq[SchemaField] = {
     sparkFields.map({ field =>
-      val arr = if (field.dataType.isInstanceOf[ArrayType]) {
-        Some(field.dataType.asInstanceOf[ArrayType])
-      } else None
+      val arr = field.dataType match {
+        case arrayType: ArrayType =>
+          Some(arrayType)
+        case _ => None
```

Comment:
Don't we usually align the arrows (`=>`) and keep them one-liners since they are short